docs/nodes/itn-enable-telemetry-nodes.md

```diff
@@ -0,0 +1,33 @@
+# Restart your node with metrics flags
+
+## Description
+
+Add metrics flags to your node start command
+and restart your node to apply it.
+
+The metrics endpoint will gather your node's data
+to track your uptime.
+
+## Directions
+
+1. Add `--metrics.tls=false --metrics --metrics.endpoint http://otel.celestia.tools:4318`
+  to your node's start command and restart your node.
+
+## Judging Criteria
+
+1. Full points if we can see data from your Celestia Node
+2. No points if we cannot
+3. Note that we will keep track of your metrics throughout the whole
+  ITN so make sure you leave it connected throughout all phases.
+
```

Comment:
I would add point 4 here:

4. Make sure you can see your node ID on [https://tiascan.com](https://tiascan.com/) in your respective node category. You can also check your node uptime status there.